superbench/benchmarks/micro_benchmarks/gpu_burn_test.py

```diff
@@ -66,12 +66,8 @@ def _preprocess(self):
         if self._args.tensor_core:
             command += ' -tc'
         command += ' {} '.format(self._args.time)
-        # copy compare.ptx which needs to be in the working directory
-        compare_copy = 'cp ' + self._args.bin_dir + '/compare.ptx ./'
-        # remove compare.ptx from working directory
-        compare_rm = 'rm ' + 'compare.ptx'
 
-        self._commands.append(compare_copy + ' && ' + command + ' && ' + compare_rm)
+        self._commands.append(command)
```

Comment:
do you need to change unit tests accordingly?

Reply:
thanks, update